lib/draper/test_case.rb

```diff
@@ -29,14 +29,10 @@ module Behavior
   end
 end
 
-if defined?(ActionController::TestCase)
-  class ActionController::TestCase
-    include Draper::TestCase::ViewContextTeardown
-  end
+class ActionController::TestCase
+  include Draper::TestCase::ViewContextTeardown
 end
 
-if defined?(ActionMailer::TestCase)
```

Comment:
May I ask why you are making this change? It breaks testing apps without ActionMailer included, right?

```
NameError: uninitialized constant ActionMailer
  <hidden>/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/bundler/gems/draper-de70008a5e68/lib/draper/test_case.rb:36:in `<top (required)>'
  <hidden>/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/bundler/gems/draper-de70008a5e68/lib/draper/railtie.rb:21:in `block in <class:Railtie>'
  etc..
```